ChameleonForms/Component/Section.cs

```diff
@@ -1,16 +1,36 @@
-﻿using System.Web;
+﻿using System;
+using System.Linq.Expressions;
+using System.Web;
 using System.Web.Mvc;
+using System.Web.Mvc.Html;
 using ChameleonForms.Component.Config;
-using ChameleonForms.Templates;
+using JetBrains.Annotations;
 
 namespace ChameleonForms.Component
 {
     /// <summary>
+    /// Interface for a modeless cast of a ChameleonForms Section.

```

Comment:
Unfortunately, some tagging interfaces are required to deal with the generics.

The generics are a curse and saviour in this library. It makes all the code more complex, but it gives us amazing intellisense everywhere as a user...